<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera_play</class>
 <widget class="QWidget" name="Camera_play">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>541</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>541</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>摄像头监控和拍照</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="open_camera_bt">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>开启摄像头</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="close_camera_bt">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>关闭摄像头</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="capture_bt">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>抓图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_video_bt">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>开始录像</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_capture_bt">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>停止录像</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_bt">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>摄像头参数</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>321</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>摄像头设备</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboCamera">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>29</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>541</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="save_pic_file_bt">
           <property name="minimumSize">
            <size>
             <width>212</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>212</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>视频保存文件</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="save_picfile_pathEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>29</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>录制时间</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>230</height>
          </size>
         </property>
         <property name="title">
          <string>摄像头预览</string>
         </property>
         <widget class="QCameraViewfinder" name="camera_widget" native="true">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>17</y>
            <width>321</width>
            <height>271</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="title">
          <string>抓取图片</string>
         </property>
         <widget class="QLabel" name="capture_label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>321</width>
            <height>271</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>摄像头状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="state_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>抓取模式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="capture_mode_label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>图片保存为：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pic_save_label">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header location="global">qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
